python/pyarrow/parquet.py

```diff
@@ -391,7 +391,13 @@ def _sanitize_table(table, new_schema, flavor):
     meaning for each codec, so you have to read the documentation of the
     codec you are using.
     An exception is thrown if the compression codec does not allow specifying
-    a compression level."""
+    a compression level.
+use_byte_stream_split: bool or list, default False
+    Specify if the byte_stream_split encoding should be used in general or
+    only for some columns. If both dictionary and byte_stream_stream are
+    enabled, then dictionary is prefered.
```

Comment:
What does "prefered" mean here? Is dictionary tried until a certain cardinality and then it falls back on bytestream splitting?

Reply:
Dictionary will be selected regardless of whether byte_stream_split is enabled. The reason for this is that byte_stream_split actually only makes sense for high entropy data (like ~15 bits and higher per element), so one should be careful with selecting it.
That's also currently the behavior in Parquet-MR but AFAIR the dictionary would fallback to byte_stream_split if it turns out that the dictionary does poorly according to some heuristic.

To use byte_stream_split one would have to explicitly disable dictionary and enable byte_stream_split.